### PR TITLE
[Bug][force-success] force success add end time

### PR DESCRIPTION
**Pull request by sdhzwc**
## Purpose of the pull request

fix:#15142
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by EricGao888 — APPROVED** [review]

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskInstanceServiceImpl.java [hunk 1]
```diff
@@ -258,6 +258,7 @@ public Result forceTaskSuccess(User loginUser, long projectCode, Integer taskIns
 
         // change the state of the task instance
         task.setState(TaskExecutionStatus.FORCED_SUCCESS);
+        task.setEndTime(new Date());
```
Comment (EricGao888) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskInstanceServiceImpl.java:258]: Could u plz add a corresponding unit test case to cover this patch? Thanks

**Review by EricGao888 — DISMISSED** [review]

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskInstanceServiceImpl.java (hunk 1 above)
Comment (sdhzwc) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskInstanceServiceImpl.java:258]: > Could u plz add a corresponding unit test case to cover this patch? Thanks


Forced success only results in a lack of end time, which prevents the runtime from terminating,I don't think this unit test is necessary，Thanks : )

**Review by qingwli — APPROVED** [review]

**Review by fuchanghai — APPROVED** [review]
LGTM

**Review by ruanwenjun — COMMENTED** [review]
Thanks for your PR, this pr basically LGTM, could you use English to describe this problem in issue #15142. And UT is needed.

**Review by rickchengx — APPROVED** [review]
It seems that this pr has been stale for several months, I would like to add related UT in another pr after it is merged.

**Pull request merged**